test/metabase/api/dashboard_test.clj

```diff
@@ -111,9 +111,8 @@
                  (update :entity_id boolean)
                  (update :collection_id boolean)
                  (update :collection boolean)
-                 (cond->
-                  (:param_values dashboard)
-                  (update :param_values not-empty)))]
+                 (cond-> (:param_values dashboard)
+                   (update :param_values not-empty)))]
```

Comment:
this is very weird, because first argument here comes from `->` up there, so this new formatting is incorrect with regards to logic :)

Reply:
yea lol, maybe cljfmt need to be smarter about this